### DISCUSSION TITLE
lazpaint in Debian

### DISCUSSION THREAD

**Discussion by alexmyczko** [bgrabitmap/lazpaint#546] [General]
Hi @circular17 

Would you mind moving lazpaint Vcs git to salsa.debian.org ? I can do that, you don't have to do anything.

Best,

**Comment by circular17** [comment]
Hello Alex,

Hope you're alright.

I am used to GitHub though I can live with a change.

Though I tried opening the salsa repository with GitKraken and it doesn't accept using the free version because it says the repository is private or self-hosted. I suppose I could afford the small subscription or try to find another free Git client on MacOS.

What are the advantages in moving the repository?

Regards

**Comment by alexmyczko** [comment]
Thank you, I am. Hope you as well...

Please you would keep using github for development. The salsa vcs (debian/control) entry is only for the debian/ packaging part.

You wouldn't use the salsa git, you'd keep github for development.

The point is the QA page lists the Vcs fields:
https://qa.debian.org/developer.php?login=circular%40fastmail.com
[Git](https://github.com/bgrabitmap/lazpaint-upstream)
[7.2.2-1](https://qa.debian.org/cgi-bin/vcswatch?package=lazpaint)

And it would allow to have an icon in the qa page. as well as be green there if we would put the packaging part there, I can give you access once you tell me what your salsa.debian.org account is if you want.

btw https://qa.debian.org/popcon.php?package=lazpaint qt gains traction. any news on the gtk2 front? gtk3 or gtk4?

Regards,
